docs/page.js

```diff
@@ -106,16 +106,22 @@ function onDocumentLoad() {
 
 	// handle code snippets formatting
 
-	const elements = document.getElementsByTagName( 'code' );
+	for ( const el of document.getElementsByTagName( 'code' ) ) {
 
-	for ( let i = 0; i < elements.length; i ++ ) {
```

Comment:
Can you please retain the canonical for loops in your PRs^^?

We use the because of their slightly better performance and we want to avoid to much of a mix.

Reply:
good